styles/style.css

```diff
@@ -3,3 +3,200 @@
+  background-color: #EAEAEA;
+  margin: 22px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 15px;
+}
+
+.aboutus-section .item {
+  width: 424px;
+  height: 474px;
+  margin: 22px;
+  display: flex;
+  justify-content: center;
+  border-radius: 15px;
+}
+
+.aboutus-section .button {
+  width: 160px;
+  height: 60px;
+  border-radius: 30px;
+  background: none;
+  font-size: 17px;
+  border: 1px solid #282828;
+}
+
+.aboutus-section .button2 {
+  width: 160px;
+  height: 60px;
+  margin-left: 40px;
+  margin-top: 160px;
+  border-radius: 30px;
+  font-size: 17px;
+  border: 1px solid #282828;
+  background: none;
+}
+
+.aboutus-section .button3 {
+  position: absolute;
+  width: 160px;
+  height: 60px;
+  margin-top: 370px;
+  margin-right: 140px;
+  font-size: 17px;
+  border: 1px solid #ffff;
+  color: #ffff;
+  border-radius: 30px;
+  background: none;
+}
+
+.aboutus-section .button4 {
+  position: absolute;
+  width: 160px;
+  height: 60px;
+  margin-top: 370px;
+  margin-right: 140px;
+  font-size: 17px;
+  border: 1px solid #FE6839;
+  color: #ffff;
+  border-radius: 30px;
+  background: none;
+}
+
+.aboutus-section .about {
+  margin-top: 35px;
+  margin-left: 100px;
+  color: #9e9898;
+}
+
+.aboutus-section .title {
+  margin-top: 30px;
+  color: #282828;
+}
+
+.aboutus-section .titletwo {
+  margin-top: 40px;
+  margin-left: 50px;
+  font-size: 30px;
+  width: 285px;
+  height: 167px;
+  color: #282828;
+}
+
+.aboutus-section .titlethere {
+  position: absolute;
+  width: 300px;
+  margin-top: 250px;
+  font-size: 35px;
+}
+
+.aboutus-section .titlefour {
+  position: absolute;
+  width: 300px;
+  margin-top: 50px;
+  font-size: 35px;
+  color: #ffff;
+}
+
+.aboutus-section .cizgi {
+  border: 1px solid gray;
+  margin-top: 35px;
+  width: 284px;
+}
+
+.aboutus-section .cizgi2 {
+  position: absolute;
+  height: 100px;
+  border: 1px solid #ffff;
+  margin-top: 160px;
+  margin-right: 300px;
+}
+
+.aboutus-section .div-two {
+  width: 900px;
+  font-size: 30px;
+}
+
+.aboutus-section .div-there {
+  margin-left: 260px;
+}
+
+.aboutus-section .div-left {
+  background-color: #EAEAEA
+}
+
+.aboutus-section .div-middle {
+  background-color: #FE6839
+}
+
+.aboutus-section .divfoto {
+  width: 106px;
+  height: 106px;
+  margin-top: 150px;
+}
+
+.aboutus-section .divfoto2 {
+  width: 400px;
+  height: 280px;
+  margin-top: 100px;
+  margin-left: -20px;
+}
+
+.aboutus-section .divfoto3 {
+  width: 78px;
+  height: 78px;
+  position: absolute;
+  margin-top: 50px;
+  margin-right: 270px;
+}
+
+.aboutus-section .divfoto4 {
+  margin-left: 20px;
+}
+
+.aboutus-section .divfoto5 {
+  border-radius: 15px;
+}
+
+.aboutus-section .divfoto6 {
+  position: absolute;
+  border-radius: 15px;
+}
+
+.aboutus-section .backfoto {
+  position: absolute;
+  top: 355px;
+  left: 0px;
+  width: 470px;
+  z-index: -2;
+}
+
+.aboutus-section .backfoto2 {
+  position: absolute;
+  top: 830px;
+  right: 0px;
+  width: 900px;
+  z-index: -2;
+}
```

Comment:
.aboutus-section .container class'ına position relative verdikten sonra, backfoto için top: 0, backfoto2 için top yerine bottom 0 vermen gerekiyor